safeeyes/__main__.py

```diff
@@ -64,29 +65,63 @@ def __running():
     try:
         # locale.bindtextdomain is required for Glade files
-        locale.bindtextdomain('safeeyes', utility.LOCALE_PATH)
+        locale.bindtextdomain("safeeyes", utility.LOCALE_PATH)
     except AttributeError:
-        logging.warning('installed python\'s gettext module does not support locale.bindtextdomain. locale.bindtextdomain is required for Glade files')
-
+        logging.warning(
+            "installed python's gettext module does not support locale.bindtextdomain."
+            " locale.bindtextdomain is required for Glade files"
+        )
 
-    parser = argparse.ArgumentParser(prog='safeeyes', description=_('description'))
+    parser = argparse.ArgumentParser(prog="safeeyes", description=_("description"))
     group = parser.add_mutually_exclusive_group()
-    group.add_argument('-a', '--about', help=_('show the about dialog'), action='store_true')
-    group.add_argument('-d', '--disable', help=_('disable the currently running safeeyes instance'), action='store_true')
-    group.add_argument('-e', '--enable', help=_('enable the currently running safeeyes instance'), action='store_true')
-    group.add_argument('-q', '--quit', help=_('quit the running safeeyes instance and exit'), action='store_true')
-    group.add_argument('-s', '--settings', help=_('show the settings dialog'), action='store_true')
-    group.add_argument('-t', '--take-break', help=_('Take a break now').lower(), action='store_true')
-    parser.add_argument('--debug', help=_('start safeeyes in debug mode'), action='store_true')
-    parser.add_argument('--status', help=_('print the status of running safeeyes instance and exit'), action='store_true')
-    parser.add_argument('--version', action='version', version='%(prog)s ' + SAFE_EYES_VERSION)
+    group.add_argument(
+        "-a", "--about", help=_("show the about dialog"), action="store_true"
+    )
+    group.add_argument(
+        "-d",
+        "--disable",
+        help=_("disable the currently running safeeyes instance"),
+        action="store_true",
+    )
+    group.add_argument(
+        "-e",
+        "--enable",
+        help=_("enable the currently running safeeyes instance"),
+        action="store_true",
+    )
+    group.add_argument(
+        "-q",
+        "--quit",
+        help=_("quit the running safeeyes instance and exit"),
+        action="store_true",
+    )
+    group.add_argument(
+        "-s", "--settings", help=_("show the settings dialog"), action="store_true"
+    )
+    group.add_argument(
+        "-t", "--take-break", help=_("Take a break now").lower(), action="store_true"
+    )
+    parser.add_argument(
+        "--debug", help=_("start safeeyes in debug mode"), action="store_true"
+    )
+    parser.add_argument(
+        "--status",
+        help=_("print the status of running safeeyes instance and exit"),
+        action="store_true",
+    )
+    parser.add_argument(
+        "--version", action="version", version="%(prog)s " + SAFE_EYES_VERSION
```

Comment:
It would be better to forbid `+` string formatting in linter rules

Reply:
I agree - however, this PR is big enough as-is (with the associated merge conflicts...).
I would prefer to leave this for a future PR, after this has landed.